docset/winserver2022-ps/defender/Get-MpPerformanceReport.md

```diff
@@ -0,0 +1,331 @@
+---
+description: Use this topic to analyze and report on Microsoft Defender Antivirus (MDAV) performance recordings.
+external help file: MSFT_MpThreat.cdxml-help.xml
+Module Name: Defender
+ms.date: 03/20/2022
+online version: 
+schema: 2.0.0
+title: Get-MpPerformanceReport
+
+---
+
+# Get-MpPerformanceReport
+
+The following section describes the Get-MpPerformanceReport PowerShell cmdlet. Analyzes and reports on Microsoft Defender Antivirus (MDAV) performance recording.
+
+## Syntax: Get-MpPerformanceReport
+
+```powershell
+Get-MpPerformanceReport    [-Path] <String>
+[-TopScans <Int32>]
+[-TopFiles  <Int32>
+    [-TopScansPerFile <Int32>]
+    [-TopProcessesPerFile  <Int32>  
+        [-TopScansPerProcessPerFile <Int32>]
+    ]
+] 
+[-TopExtensions  <Int32>
+    [-TopScansPerExtension <Int32>]
+    [-TopProcessesPerExtension <Int32>
+        [-TopScansPerProcessPerExtension <Int32>]
+        ]
+    [-TopFilesPerExtension  <Int32>
+        [-TopScansPerFilePerExtension <Int32>]
+        ]
+    ] 
+]
+[-TopProcesses  <Int32>
+    [-TopScansPerProcess <Int32>]
+    [-TopExtensionsPerProcess <Int32>
+        [-TopScansPerExtensionPerProcess <Int32>]
+    ]
+]
+[-TopFilesPerProcess  <Int32>
+    [-TopScansPerFilePerProcess <Int32>]
+]
+[-MinDuration <String>]
+```
+
+## Description: Get-MpPerformanceReport
```

Comment:
Cannot change the H2. The label "Get-MpPerformanceReport" is redundant and not needed anyway.